server/database/database.js

```diff
@@ -1,6 +1,6 @@
 const mongoose = require('mongoose');
 require('dotenv').config();
-mongoose.connect(process.env.localDB);
+mongoose.connect('mongodb://127.0.0.1:27017');
```

Comment:
Do you know how these are supposed to work? You can create a file called `.env` in the root folder (next to package.json and such), then assign that same string (without quotation marks) to `localDB`. We'll have to change this before it's deployed or the DB will immediately be taken over by bots.

Reply:
I revised it.